src/server.js

```diff
@@ -1,11 +1,18 @@
-import app from './app';
+import app from '../src/app';
 import config from './config/config';
 
+
 const currentConfig = config[process.env.NODE_ENV];
 
 const { port } = currentConfig;
 // console.log(port);
 
+
+
+const {port} = config;
```

Comment:
Remove line 12 because the port is already declared at line 7 that is causing the error when you start the server. Also remove the commented line 7.